src/main/java/ch/sportchef/business/event/boundary/EventImageResource.java

```diff
@@ -102,6 +109,9 @@ public Response uploadImage(@Context HttpServletRequest request) throws IOExcept
                 }
             }
         }
+        BufferedImage plainImage = ImageIO.read(file);
```

Comment:
Here is a potential `NullPointerException` on line 112. Caused by initializing the `file` object with `null` on line 98 and creating the `File` object in a `while` loop on line 104 which must not be executed (`while (nextPart)` could evaluate to `false`). Checking `file` for `null` before accessing the object should solve this `NullPointerException`.